wrapper/src/test/java/integration/util/AuroraTestUtility.java

```diff
@@ -923,4 +924,20 @@ public void addAuroraAwsIamUser(
       }
     }
   }
+
+  public void updateInstanceCertificateIdentifier(
+      String dbInstanceIdentifier, String certificateIdentifier) {
+    try {
+      ModifyDbInstanceRequest modifyDbInstanceRequest = ModifyDbInstanceRequest.builder()
+          .dbInstanceIdentifier(dbInstanceIdentifier)
+          .publiclyAccessible(true)
+          .applyImmediately(true)
+          .caCertificateIdentifier(certificateIdentifier)
+          .build();
+      rdsClient.modifyDBInstance(modifyDbInstanceRequest);
+    } catch (Exception e) {
```

Comment:
If this exception occurs, should the test fail? Does it need to be re thrown?

Reply:
It should now re throw and fail the test for an exception